.changes/next-release/bugfix-dynamodb-38777.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "dynamodb",
```

Comment:
I think we got a stray changelog entry

Reply:
Correct, will clean that up with the next push that's actually meant for this branch, to avoid unnecessary GH Actions runs.